### PR TITLE
Patch/72 new test data set for ehrbase/project management#72

### PR DESCRIPTION
**Pull request by jakesmolka**
## Changes

-  Adds first contribution-context checks
    - Adds check if a version with change type CREATION also has preceding_version_uid set, or with type MODIFICATION has the version uid not set. Both are invalid. 
- Also fixed integration test for the first case.
- Small change to existing test data

## Related issue

fixes https://github.com/ehrbase/project_management/issues/72

## Additional information and checks

- [x] Pull request linked in changelog

### CODE REVIEW

**Comment by testautomation** [comment]
quick command to run related test (on this branch!)
```
robot -i 72 -d results -L TRACE robot/CONTRIBUTION_TESTS
```

**Comment by jakesmolka** [comment]
(added to current sprint - was not transferred from last to current from the beginning)

File: service/src/main/java/org/ehrbase/service/ContributionServiceImp.java [hunk 1]
```diff
@@ -187,11 +190,45 @@ private void processCompositionVersion(UUID ehrId, UUID contributionId, Version
...
+                if (version.getPrecedingVersionUid() != null)
+                    throw new InvalidApiParameterException("Invalid version. Change type CREATION, but also set \"preceding_version_uid\" attribute");
+                break;
+            case MODIFICATION:
+                // can't have change type MODIFICATION and without giving "preceding_version_uid"
+                if (version.getPrecedingVersionUid() == null)
+                    throw new InvalidApiParameterException("Invalid version. Change type MODIFICATION, but without \"preceding_version_uid\" attribute");
+                break;
+            // block of valid change types, without any rules to apply (yet)
+            case AMENDMENT:
```
Comment (ppazos) [inline service/src/main/java/org/ehrbase/service/ContributionServiceImp.java:221]: @jakesmolka amendment and modification have the same logic

Comment (jakesmolka) [reply service/src/main/java/org/ehrbase/service/ContributionServiceImp.java]: Thanks! Fixed in https://github.com/ehrbase/ehrbase/pull/202/commits/ace3e3abdd6ee8da505f49d6a6da87f994d55f45

**Pull request merged**